app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,65 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.model.Task;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//TODO
+//1. Read Collection - GET/tasks
+//2. Read Item - GET/tasks/{id}
+//3. Create - POST/tasks
+//4. Update - PUT/PATCH /tasks/{id}
+//5. Delete - DELETE /tasks/{id}
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private List<Task> tasks = new ArrayList<>();
+    private Long newId = 0L;
+
+    @GetMapping
+    public List<Task> list(){
+
+        return tasks;
+    }
+    @PostMapping
```

Comment:
IDE에서 제공하는 포매팅 기능을 조사해 보시고, 단축키를 사용해 포매팅을 해보세요. 띄어쓰기 등에 대해서 신경쓸 일이 사라질 거에요.

```suggestion
    @GetMapping
    public List<Task> list(){
        return tasks;
    }

    @PostMapping
```